tests/examples_test.py

```diff
@@ -0,0 +1,30 @@
+from pathlib import Path
+
+import pytest
+
+from blackboxopt.examples import multi_objective_multi_param
+
+
+@pytest.mark.parametrize("example_module", [multi_objective_multi_param])
```

Comment:
Good idea to test the examples! Would you add the other example (dash distributed) in a separate ticket/PR, or would it be easier to include that here in one go?

Reply:
Since it requires a slight rewrite and the PR was already larger than I like, I wanted to handle it separately.